### ISSUE TITLE
error building wheels for 3.8

### ISSUE THREAD

**Issue by Zomatree** [discord/erlpack#30]
```py
PS C:\Users\Zoomatree\Documents\GitHub\erlpack-test> pip install erlpack
Collecting erlpack
  Downloading erlpack-0.3.3.tar.gz (124 kB)
     |████████████████████████████████| 124 kB 1.1 MB/s

Requirement already satisfied: six==1.12.0 in c:\users\zoomatree\.virtualenvs\erlpack-test-0lyfi90o\lib\site-packages (from erlpack) (1.12.0)
Building wheels for collected packages: erlpack
  Building wheel for erlpack (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: 'C:\Users\Zoomatree\.virtualenvs\erlpack-test-0LyfI90o\Scripts\python.exe' -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\Zoomatree\\AppData\\Local\\Temp\\pip-install-enskivqz\\erlpack\\setup.py'"'"'; __file__='"'"'C:\\Users\\Zoomatree\\AppData\\Local\\Temp\\pip-install-enskivqz\\erlpack\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d 'C:\Users\Zoomatree\AppData\Local\Temp\pip-wheel-8fdu7kbb'
       cwd: C:\Users\Zoomatree\AppData\Local\Temp\pip-install-enskivqz\erlpack\
  Complete output (24 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build\lib.win-amd64-3.8
  creating build\lib.win-amd64-3.8\erlpack
  copying py\erlpack\constants.py -> build\lib.win-amd64-3.8\erlpack
  copying py\erlpack\types.py -> build\lib.win-amd64-3.8\erlpack
  copying py\erlpack\__init__.py -> build\lib.win-amd64-3.8\erlpack
  warning: build_py: byte-compiling is disabled, skipping.

  running build_ext
  building 'erlpack._packer' extension
  creating build\temp.win-amd64-3.8
  creating build\temp.win-amd64-3.8\Release
  creating build\temp.win-amd64-3.8\Release\py
  creating build\temp.win-amd64-3.8\Release\py\erlpack
  C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Tools\MSVC\14.16.27023\bin\HostX86\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -IC:\Users\Zoomatree\.virtualenvs\erlpack-test-0LyfI90o\include -Ic:\python38\include -Ic:\python38\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Tools\MSVC\14.16.27023\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\cppwinrt" /EHsc /Tppy/erlpack/_packer.cpp /Fobuild\temp.win-amd64-3.8\Release\py/erlpack/_packer.obj -O3
  cl : Command line warning D9002 : ignoring unknown option '-O3'
  _packer.cpp
  py/erlpack/_packer.cpp(4918): warning C4146: unary minus operator applied to unsigned type, result still unsigned
  py/erlpack/_packer.cpp(7109): error C2660: 'PyCode_New': function does not take 16 arguments
  c:\python38\include\code.h(122): note: see declaration of 'PyCode_New'
  error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\BuildTools\\VC\\Tools\\MSVC\\14.16.27023\\bin\\HostX86\\x64\\cl.exe' failed with exit status 2
  ----------------------------------------
  ERROR: Failed building wheel for erlpack
  Running setup.py clean for erlpack
Failed to build erlpack
Installing collected packages: erlpack
    Running setup.py install for erlpack ... error
    ERROR: Command errored out with exit status 1:
     command: 'C:\Users\Zoomatree\.virtualenvs\erlpack-test-0LyfI90o\Scripts\python.exe' -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\Zoomatree\\AppData\\Local\\Temp\\pip-install-enskivqz\\erlpack\\setup.py'"'"'; __file__='"'"'C:\\Users\\Zoomatree\\AppData\\Local\\Temp\\pip-install-enskivqz\\erlpack\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\Zoomatree\AppData\Local\Temp\pip-record-ay0w_vkn\install-record.txt' --single-version-externally-managed --compile --install-headers 'C:\Users\Zoomatree\.virtualenvs\erlpack-test-0LyfI90o\include\site\python3.8\erlpack'
         cwd: C:\Users\Zoomatree\AppData\Local\Temp\pip-install-enskivqz\erlpack\
    Complete output (24 lines):
    running install
    running build
    running build_py
    creating build
    creating build\lib.win-amd64-3.8
    creating build\lib.win-amd64-3.8\erlpack
    copying py\erlpack\constants.py -> build\lib.win-amd64-3.8\erlpack
    copying py\erlpack\types.py -> build\lib.win-amd64-3.8\erlpack
    copying py\erlpack\__init__.py -> build\lib.win-amd64-3.8\erlpack
    warning: build_py: byte-compiling is disabled, skipping.

    running build_ext
    building 'erlpack._packer' extension
    creating build\temp.win-amd64-3.8
    creating build\temp.win-amd64-3.8\Release
    creating build\temp.win-amd64-3.8\Release\py
    creating build\temp.win-amd64-3.8\Release\py\erlpack
    C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Tools\MSVC\14.16.27023\bin\HostX86\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -IC:\Users\Zoomatree\.virtualenvs\erlpack-test-0LyfI90o\include -Ic:\python38\include -Ic:\python38\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Tools\MSVC\14.16.27023\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\cppwinrt" /EHsc /Tppy/erlpack/_packer.cpp /Fobuild\temp.win-amd64-3.8\Release\py/erlpack/_packer.obj -O3
    cl : Command line warning D9002 : ignoring unknown option '-O3'
    _packer.cpp
    py/erlpack/_packer.cpp(4918): warning C4146: unary minus operator applied to unsigned type, result still unsigned
    py/erlpack/_packer.cpp(7109): error C2660: 'PyCode_New': function does not take 16 arguments
    c:\python38\include\code.h(122): note: see declaration of 'PyCode_New'
    error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\BuildTools\\VC\\Tools\\MSVC\\14.16.27023\\bin\\HostX86\\x64\\cl.exe' failed with exit status 2
    ----------------------------------------
ERROR: Command errored out with exit status 1: 'C:\Users\Zoomatree\.virtualenvs\erlpack-test-0LyfI90o\Scripts\python.exe' -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\Zoomatree\\AppData\\Local\\Temp\\pip-install-enskivqz\\erlpack\\setup.py'"'"'; __file__='"'"'C:\\Users\\Zoomatree\\AppData\\Local\\Temp\\pip-install-enskivqz\\erlpack\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\Zoomatree\AppData\Local\Temp\pip-record-ay0w_vkn\install-record.txt' --single-version-externally-managed --compile --install-headers 'C:\Users\Zoomatree\.virtualenvs\erlpack-test-0LyfI90o\include\site\python3.8\erlpack' Check the logs for full command output.
```

**Comment by Zomatree** [comment]
peepeee

**Comment by oliver-ni** [comment]
I'm having the same issue. Does anyone have a workaround?

**Comment by birarda** [comment]
This is fixed in main branch now thanks to #49. We will publish a new package shortly.
